pydra/utils/tests/utils.py

```diff
@@ -17,6 +20,34 @@ class MyFormatX(WithSeparateHeader, MyFormat):
     header_type = MyHeader
 
 
+class MyOtherFormatX(WithMagicNumber, WithSeparateHeader, File):
```

Comment:
is this only used in tests?

Reply:
Yes, but note it is under the `pydra.utils`' `tests` directory not the package proper